apps/taxes/src/modules/avatax/ui/edit-avatax-configuration.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { AvataxConfigurationForm } from "./avatax-configuration-form";
+import { AvataxConfig } from "../avatax-config";
+import { trpcClient } from "../../trpc/trpc-client";
+import { useDashboardNotification } from "@saleor/apps-shared";
+import { useRouter } from "next/router";
+import { z } from "zod";
+
+export const EditAvataxConfiguration = () => {
+  const router = useRouter();
+  const { id } = router.query;
+  const configurationId = z.string().parse(id);
+
+  const { refetch: refetchProvidersConfigurationData } =
+    trpcClient.providersConfiguration.getAll.useQuery();
+
+  const { notifySuccess, notifyError } = useDashboardNotification();
+  const { mutate: patchMutation, isLoading: isPatchLoading } =
+    trpcClient.avataxConfiguration.patch.useMutation({
+      onSuccess() {
+        notifySuccess("Success", "Updated Avatax configuration");
+        refetchProvidersConfigurationData();
+      },
+      onError(error) {
+        notifyError("Error", error.message);
+      },
+    });
+
+  const { data } = trpcClient.avataxConfiguration.get.useQuery({ id: configurationId });
```

Comment:
as mention above - if data doesnt exist and its loading - render some placeholder instead of form

if loading is done and data doesn't exist - it should be handled (error boundary or something), it also should go to Sentry (this should never happen)